storybookjs/storybook
# Docs: Make webpackFinal consistent across docs

## What I did

Makes the implementation of `webpackFinal` consistent across documentation.

In the documentation there is both,

```
webpackFinal: async (...
```

and,

```
async webpackFinal(...
```

While these are both valid, I feel the documentation is easier to read if implementation is consistent?

I've aligned these cases to the style that is used elsewhere.

## Checklist for Contributors

### Testing

#### The changes in this PR are covered in the following automated tests:

None that I'm aware of?

#### Manual testing

I'm not sure how to run the docs locally? Apologies if I've missed that somewhere. Testing should be as follows:

- running the docs locally
- go to the local equivalent of https://storybook.js.org/docs/7.5/builders/webpack
- check that  the implementation is now consistent


## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
